<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>串口</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_com">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_baud">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_openPort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_searchPort">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>测试按钮</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>相机</string>
    </property>
    <widget class="QPushButton" name="pushButton_OpenCam">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开相机</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_channel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>651</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>图像显示</string>
    </property>
    <widget class="QLabel" name="label_original">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px;
border-style: outset;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_gray">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px;
border-style: outset;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_binary">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px;
border-style: outset;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>原图：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>灰度图：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>二值化：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
